FV3/gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -4496,6 +4550,26 @@ subroutine GFS_physics_driver                                 &
                         Stateout%gq0(1,1,1), Stateout%gq0(1,1,ntcw),           &
                         Stateout%gt0, rain1, Diag%sr, rainp, rhc, psautco_l,   &
                         prautco_l, Model%evpco, Model%wminco, lprnt, ipr)
+            
+#ifdef ENABLE_CALLPYFORT
+            do k=1,levs
+              do i=1,im
+                qv_post_precpd(i,k) = Stateout%gq0(i,k,1)
+                qc_post_precpd(i,k) = Stateout%gq0(i,k,ntcw)
+              enddo
+            enddo
+
+            call set_state("air_temperature_output", Stateout%gt0)
+            call set_state("specific_humidity_output", qv_post_precpd)
+            call set_state("cloud_water_mixing_ratio_output", qc_post_precpd)
+
+            call set_state("total_precipitation", rain1)
+            call set_state("ratio_of_snowfall_to_rainfall", Diag%sr)
+            call set_state("tendency_of_rain_water_mixing_ratio_due_to_microphysics", rainp)
+            call call_function("emulation.monitor", "store_zarr")
+            call call_function("emulation.monitor", "store_netcdf")
```

Comment:
I don't see any `get_state` commands. Is your code for actually calling the NN in another branch? Maybe we should have a namelist option that controls the `mode` of the call_py_fort emulation e.g.:

1. emulation.save_data = .true.
2. emulation.online = .true.

Reply:
Yes, the get state is still on the development branch I was using for testing prognostic results.
https://github.com/VulcanClimateModeling/fv3gfs-fortran/tree/andrep/microphys_emu_online

I like that idea for the flags.  Wasn't sure if we wanted to go the compilation route for switching between behaviors.  Namelist flags makes it a lot easier on having these all available in the wrapper image without having to adjust the dockerfile.

Reply:
This may be more food for thought. We can generalize this code once you merge that branch.